Create AddRemoveCartTest

Petr-Kv opened the pull request:
I look forward to your suggestions for improving the code, I will be glad to any comments.

Petr-Kv committed:
Create AddRemoveCartTest

I look forward to your suggestions for improving the code, I will be glad to any comments.

sabinawozniak commented on src/test/java/org/example/AddRemoveCartTest:
```diff
@@ -0,0 +1,94 @@
+    public void addCart() {
+        System.setProperty(
+                "webdriver.chrome.driver",
+                "D:\\Git\\practicMaventest1\\src\\test\\resources\\webdriver\\chromedriver.exe");
```
try to use dependency from today's lesson. this might not work on another computer.

Petr-Kv replied:
fixed

mat-kud commented:
Use nouns to name variables, not verbs. Verbs are reseved for methods. Also try to describe elements more precisely. Variable names like "addInformationLabel" or "addInformationLabel2" dont tell you much. Try to get rid of repetitive code by using @BeforeTest or @BeforeMethod

shibiza commented:
Hello! You may try to use not just @Test annotation, but also @BeforeMethod and @AfterMethod. Just have a look as I do it at my branch https://github.com/shibiza/group_3/tree/delivery_to_functionality

By the way as I can see you had just 2 tests (2 methods):    public void addCart()  and  public void deleteCart(), so please try to use my advise and expand your list of used annotations

shibiza commented:
and you know that's strange... but your class AddRemoveCartTest doesn't have ".java" and your code doesn't want to compilate in my Intelijy because of that. I can read your code only in github. It's better to fix that ASAP

Petr-Kv committed:
Update pom.xml

Petr-Kv committed:
Update and rename AddRemoveCartTest to Task - 3 addRemoveCartTest.java

Added @BeforeMethod and @AfterMethod, tried to name the variables more clearly and added ".java" to the filename.

mat-kud commented:
Lines #26 and #34, you still use verbs to describe variable

Petr-Kv committed:
Delete AppTest.java

delete AppTest.java

Petr-Kv committed:
Update Task - 3 addRemoveCartTest.java

changes added to lines # 26 and # 34

shibiza commented:
sorry but you did it wrong. Read carefully how to name java.classes and where to store them. It was on our lecture. None of your @Test works

Petr-Kv committed:
Rename Task - 3 addRemoveCartTest.java to AddRemoveCartTest.java

juliaHrabovska commented on src/test/java/org/example/AddRemoveCartTest.java:
```diff
@@ -0,0 +1,77 @@
+
+        WebElement checkInformationLabelCart = new WebDriverWait(webDriver, Duration.ofSeconds(10))
+                .until(ExpectedConditions.visibilityOfElementLocated(By.xpath("//*[@id=\"nav-cart-count\"]")));
+        Assert.assertEquals("1", checkInformationLabelCart.getText());
```
It would be better to add a string with the explanation of what is expected
